_stories/molecules/Table/index.js

```diff
@@ -86,7 +86,9 @@ const columnsAdvanced = [
             </ToolTip>
         ),
         disableSorting: true,
-        onHeadingClick: action('column_heading_clicked')
+        headingProps: {
+            onClick: action('column_heading_clicked')
```

Comment:
Shouldn't this log something when clicking the heading? I don't see anything on storybook's log

Reply:
I only see the `row_clicked` one

Reply:
I'm also trying to pass a classname but don't see it on the dom

Reply:
Are you using the "advanced" column config in the knobs? There's a few different configs and  these will only work when the advanced are selected. 

<img width="370" alt="screen shot 2018-06-13 at 8 53 48 pm" src="https://user-images.githubusercontent.com/7143485/41390783-0c25197e-6f4c-11e8-9dc6-43258c43d776.png">

Reply:
![oh-you-dog](https://user-images.githubusercontent.com/1082158/41424686-64914c0e-6fc4-11e8-89cf-4a757704bb97.jpg)